First_section.html

```diff
@@ -9,53 +9,276 @@
         <p id="paragraph">
           I am software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
-          <h2>LET'S CONNECT</h2>
-       <div class="icons"> 
-        <ul>
-          <li>
-            <a href="#">
-              <img class="social" src="Vector.png" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social" src="Linkedin icon.png" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social" src="Vector (1).png" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social" src="Vector (3).png" />
-            </a>
+        <h2 id="connect">LET'S CONNECT</h2>
+        <div class="icons">
+          <ul class="Image">
+            <li>
+              <a href="#">
+                <img class="social" src="Vector.png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Linkedin icon.png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Vector (1).png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Vector (3).png" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="social" src="Vector (2).png" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="work_container">
+        <div class="work works0">
+          <img src="Snapshoot Portfolio.png" alt="snashoot" />
+          <div class="Project info">
+            <h1 class="project_name">Tonic</h1>
+            <ul class="client">
+              <li id="h3">CANOPY</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="langue">Back end Dev</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="year">2015</li>
+            </ul>
+            <p class="project_description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="code_language">
+              <li class="html"><a class="ht" href="#">html</a></li>
+              <li class="css"><a class="ht" href="#">css</a></li>
+              <li class="javascript"><a class="ht" href="#">javascript</a></li>
+            </ul>
+            <div class="link_to_project">
+              <li class="check_Project"><a href="#">See Project</a></li>
```

Comment:
- [ ] Kindly remove the `li` tag wrapping this button since it's not part of a list, please use the code that the previous reviewer specified 👍 (this applies to all the buttons).

[Optional] 
- [ ] This is part of the project requirements but since the previous reviewer missed it it's marked as optional. Please consider adding the button interaction styles (hover, pressed... etc) as shown in Figma 👍

Reply:
Done thank you.